src/@types/event.ts

```diff
@@ -283,19 +298,64 @@ export const LOCAL_NOTIFICATION_SETTINGS_PREFIX = new UnstableValue(
  */
 export const UNSIGNED_THREAD_ID_FIELD = new UnstableValue("thread_id", "org.matrix.msc4023.thread_id");
 
-export interface IEncryptedFile {
-    url: string;
-    mimetype?: string;
-    key: {
-        alg: string;
-        key_ops: string[]; // eslint-disable-line camelcase
-        kty: string;
-        k: string;
-        ext: boolean;
-    };
-    iv: string;
-    hashes: { [alg: string]: string };
-    v: string;
+/**
+ * @deprecated in favour of {@link EncryptedFile}
+ */
+export type IEncryptedFile = EncryptedFile;
+
+export interface TimelineEvents {
```

Comment:
Actually: please could this (and `StateEvents`) get some documentation? What does it represent?